README.md

```diff
@@ -15,6 +22,8 @@ This module is currently only tested on:
 * CentOS/RedHat 6
 * CentOS/RedHat 7
```

Comment:
Perhaps just drop this list given the note?

Reply:
done